contracts/Everlong.sol

```diff
@@ -307,35 +299,99 @@ contract Everlong is IEverlong {
     // │ Rebalancing                                             │
     // ╰─────────────────────────────────────────────────────────╯
 
-    // TODO: Handle case where rebalancing would exceed gas limit
-    // TODO: Handle when Hyperdrive has insufficient liquidity.
+    // NOTE: Errors from hyperdrive are not handled. The keeper must configure
+    //       the correct parameters to avoid issues with insufficient liquidity
+    //       and running out of gas from mature position closures.
```

Comment:
Something else I'd ask is "do we ever want someone to use these defaults?" Assuming the transaction is MEV-protected, it's not too bad, but I feel like it would be just as easy to remove this function and force people to use the variant that supports options.

Reply:
Totally game to just leave the one with the options... will be nicer to not have the overload.